LINUX/configure

```diff
@@ -883,7 +883,7 @@ EOF
 		message " NOTE  " <<EOF
 We are trying to download the original sources for driver
 $d using the following command:
-  
+
```

Comment:
Please leave the whitespace alone, otherwise the patch touches too many unrelated lines.